### DISCUSSION TITLE
Support for multiple boards

### DISCUSSION THREAD

**Discussion by manucompiles** [BoxThatBeat/obsidian-agile-task-notes#24] [Ideas]
Hey @BoxThatBeat,

This is a cool plugin and I love the features it brings in. I would like to suggest adding the ability to configure multiple boards and multiple dashboards for the same board. I know this is easier said than done but I am looking forward to these upgrades. Would like to hear what are your plans as well.

Cheers!!
Manu

**Comment by zivoc** [comment]
I ask you to refine the functionality so that locally created cards are not deleted after updating the board.

**Comment by zivoc** [comment]
so I ask you to add a disable item to the Kanban Task list if they are not in use.
